corehq/motech/fhir/utils.py

```diff
@@ -53,3 +58,20 @@ def update_fhir_resource_property(case_property, fhir_resource_type, fhir_resour
                                                       resource_type=fhir_resource_type)
         fhir_resource_prop.jsonpath = fhir_resource_prop_path
         fhir_resource_prop.save()
+
+
+def build_capability_statement(domain):
+    """
+    Builds the FHIR capability statement including the OAuth URL extensions for SMART
+    https://hl7.org/fhir/smart-app-launch/conformance/index.html
+    """
+    with open(os.path.join(os.path.dirname(__file__), 'data', 'capability_statement.json'), encoding='utf-8') as f:
```

Comment:
I think we can just cache this template in memory to avoid the IO each time this is called?

```
@memoized
def load_capability_statement():
    return open(os.path.join(os.path.dirname(__file__), 'data', 'capability_statement.json'), encoding='utf-8').read()
```

Reply:
https://github.com/dimagi/commcare-hq/pull/29453/commits/6ccce12c009e78aa868273b1bcb4b02ce34f7e88